website/docs/docs/deploy/ci-jobs.md

```diff
@@ -25,19 +25,19 @@ To make CI job creation easier, many options on the **CI job** page are set to d
 
 1. On your deployment environment page, click **Create job** > **Continuous integration job** to create a new CI job. 
 
-1. Options in the **Job settings** section:
+2. Options in the **Job settings** section:
     - **Job name** &mdash; Specify the name for this CI job.
     - **Description** &mdash; Provide a description about the CI job.
     - **Environment** &mdash; By default, it’s set to the environment you created the CI job from. Use the dropdown to change the default setting. 
```

Comment:
```suggestion
    - **Environment** &mdash; By default, this will be set to the environment you created the CI job from. Use the dropdown to change the default setting. 
```